### ISSUE TITLE
[Feature suggestion] --wallet-dir flag

### ISSUE THREAD

**Issue by selsta** [monero-project/monero#7674]
From IRC

```
22:11 <clavi> can I set a wallet directory for monero-wallet-cli?
22:11 <clavi> the config file only lets me set a single wallet
22:13 <clavi> I want to be able to run something like "monerocli <walletname>" from any directory
22:13 <clavi> without typing the full paths each time
22:17 <clavi> would be neat to have a "--wallet-dir" option in the CLI config file
```

**Comment by trasherdk** [comment]
Or a search path for `config-file` if only a filename is given.

Something like `./filename` `~/.bitmonero/filename` `.....`

**Comment by mj-xmr** [comment]
> Or a search path for `config-file` if only a filename is given.
> 
> Something like `./filename` `~/.bitmonero/filename` `.....`

Makes sense, although `--wallet-dir` is quite a known standard originating from BTC and all its forks.

**Comment by hyc** [comment]
This option exists for monero-wallet-rpc already. Should be easy enough to add to monero-wallet-cli.

Would be even easier, to avoid duplication of effort, to undo the split and consolidate both of the programs back into a single executable.

**Comment by binex-dsk** [comment]
I think it'd be easier if it was an environment variable; e.g. `MONERO_WALLET_DIR=.local/share/monero monero-wallet-cli`, as it could be set in e.g. your `.zshrc` or `.bashrc`. I suppose having both could work though
